Makefile

```diff
@@ -56,11 +54,13 @@ build-clean: clean
 dist: dist-test
```

Comment:
There are two rules point to `dist-test`, 
```
test-dist: dist-test
dist: dist-test
```

I feel they are duplicated, if we just keep one will be more concise.